pkg/install/v1alpha1/cstor_volume_0.7.0.go

```diff
@@ -594,6 +606,28 @@ spec:
     {{- .TaskResult.readlistsvc.items | notFoundErr "target service not found" | saveIf "readlistsvc.notFoundErr" .TaskResult | noop -}}
     {{- jsonpath .JsonResult "{.items[*].spec.clusterIP}" | trim | saveAs "readlistsvc.clusterIP" .TaskResult | noop -}}
 ---
+# runTask to list cstor volume cr
+apiVersion: openebs.io/v1alpha1
+kind: RunTask
+metadata:
+  name: cstor-volume-read-listcstorvolumecr-default-0.7.0
+spec:
+  meta: |
+    id: readlistcv
+    runNamespace: {{.Config.RunNamespace.value}}
+    apiVersion: openebs.io/v1alpha1
+    kind: CStorVolume
+    action: list
+    options: |-
+      labelSelector: openebs.io/persistent-volume={{ .Volume.owner }}
+  post: |
+    {{- jsonpath .JsonResult "{.items[*].metadata.name}" | trim | saveAs "readlistcv.names" .TaskResult | noop -}}
+    {{- .TaskResult.readlistcv.names | notFoundErr "cStor Volume CR not found" | saveIf "readlistcv.notFoundErr" .TaskResult | noop -}}
+    {{- jsonpath .JsonResult "{.items[*].metadata.annotations.openebs\\.io/fs-type}" | trim | default "ext4" | saveAs "readlistcv.fsType" .TaskResult | noop -}}
```

Comment:
Is the default for backward compatibility with RC1? If not then it is not required as there will be always a default value from cast config.

Reply:
Here we are fetching fs type from annotation but CStorVolume has it in labels

Reply:
What can be done for this backward compatibility cc @kmova

Reply:
Updated the previous one to annotations. W.r.t to backward comparability - this will be used only during creating new volumes.